lib/logging/summary.js

```diff
@@ -17,6 +18,15 @@ module.exports = function summary(request, response) {
     scope: request.auth.credentials.scope
   };
 
+  var payloadKeys = [];
+  try {
+    // given payload object might not be a valid object
+    // See issue #410
+    payloadKeys = Object.keys(payload);
+  } catch (e) {
+    // failed to parse payload keys.
```

Comment:
I wonder if it's worth adding some sort of sentinel value, like `payloadKeys = ['INVALID_PAYLOAD_OBJECT']` or similar, so that we can easily identify any such cases in the logs.